keyboards/alt67/keymaps/default/keymap.c

```diff
@@ -0,0 +1,211 @@
+#include "alt67.h"
+
+#include "samd51j18a.h"
```

Comment:
Most, if not all of thes includes should be handled by quantum.h. And the ones that aren't, should be handled by the keyboard's h file (included with QMK_KEYBOARD_H)

If you could remove/move these to the keyboard h?